app/views/buyers/accounts/show.html.slim

```diff
@@ -63,11 +63,12 @@ h1 data-hook="account-show"
                                  method: :post,
                                  class: 'reject action'
             - if can? :suspend, @account
-              = action_link_to 'Suspend',
-                               suspend_admin_buyers_account_path(@account),
-                               method: :post,
-                               data: { confirm: 'Are you sure?', disable_with: 'suspending…' },
-                               class: 'action suspend'
+              - unless can? :resume, @account
```

Comment:
This is if the current_user has permission to ever `resume` the `@account`.
This is incorrect for 2 reasons:
1. What we want is, that if the `@account` has is `suspended` or scheduled_for_deletion`, do not show the Suspend link. Nothing to do with the current_user having permissions to change the state, but related to the state of this `@account` instead.
2. `resume` and `suspend` are 2 different actions (and they end in different states).